auth/options.go

```diff
@@ -50,10 +50,10 @@ func PrivateKey(key string) Option {
 }
 
 // Credentials sets the auth credentials
-func Credentials(id, secret string) Option {
+func Credentials(id, refresh string) Option {
```

Comment:
This should likely remain id, secret as auth.Credentials signals id, secret.

Reply:
We don't want to store the secret (password) ever client side. We're just using the refresh token to generate new tokens, so I don't see how to action this?

Reply:
This is a bit confusing. There is an option called Credentials. So my assumption is auth.Credentials lets me set id/secret and these will be used for something. If thats not the case this should be removed.

Reply:
I see this as relevant for say my local CLI is loading credentials from disk and then using them as part of auth or its calling the Generate/Token methods but its best to remove them to remove ambiguity.

Reply:
We need this method since we need a way to set the account ID and refresh token. What would you rather have the method called?

Reply:
This is used by cmd/config to set the services credentials for srv => srv requests. Services have no passwords, just refresh tokens.